lambda/ham_predict_updater/__init__.py

```diff
@@ -619,9 +622,16 @@ async def run_predictions_for_serial(sem, flight_docs, serial, value):
                     )
                 )
             ]
-        if _flight_doc["float_expected"]:
-            if abs(value['rate']) < ASCENT_RATE_THRESHOLD:
-                logging.debug(f"{serial} running float profile")
+
+        # The balloon is either floating, or ascending.
+
+        if "no_flight_doc" in _flight_doc:
```

Comment:
should we check that no_flight_doc == True as well? I can see this causing a bug in the future if we aren't careful

Reply:
Possibly? In this case i'm just using it as a flag to say 'we havent found a flight doc', and wasnt sure of a better way of doing that.

Reply:
I just feel like future us will do something like `_flight_doc['no_flight_doc'] = False` and then wonder why things sometimes explodes 

should be `"no_flight_doc" in _flight_doc and _flight_doc['no_flight_doc'] == True`